tools/ci_build/build.py

```diff
@@ -1888,6 +1920,7 @@ def main():
     configs = set(args.config)
 
     # setup paths and directories
+    # cmake_path and ctest_path can be None
```

Comment:
>cmake_path [](http://example.com/codeflow?start=6&length=10)

when is it valid for cmake_path to be None?

Reply:
if a person only wants to run the tests, he/she doesn't need to have cmake or ctest. I just added a comment there.